packages/geoview-core/src/core/components/footer-bar/footer-bar.tsx

```diff
@@ -19,14 +19,14 @@ const sxClasses = {
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
-    width: '100%',
-    maxHeight: '34px',
-    minHeight: '34px',
+    width: 'calc(100% - 64px)',
+    maxHeight: '35px',
+    minHeight: '35px',
     backdropFilter: 'blur(5px)',
-    backgroundColor: '#000000aa',
+    backgroundColor: '#000000cc',
     pointerEvents: 'all',
     position: 'absolute',
-    left: 0,
```

Comment:
If you are using absolute this is necessary to define position of the element. Maybe try `left: '64px';`.. or `left:0; marginLeft:64px`